conda_libmamba_solver/solver.py

```diff
@@ -182,6 +183,11 @@ def __init__(
         self.solver = None
         self._solver_options = None
 
+    @staticmethod
```

Comment:
Actually, do you think we could just decorate this with `functools.lru_cache(maxsize=None)` so we don't have to recreate it on every call in conda?